apps/web/src/app/layout.tsx

```diff
@@ -31,7 +37,7 @@ export default function RootLayout({
         <meta name='viewport' content='width=device-width, initial-scale=1' />
         <link rel='icon' href='/favicon.svg' />
       </head>
-      <body className={`${fontSans.variable} ${fontMono.variable}`}>
+      <body className={`${fontSans.variable} ${fontMono.variable} font-sans`}>
```

Comment:
Why `font-sans` here?

Reply:
Otherwise any text that does not use this class would not be assigned any font, and would just use the browser's fallback.

An example where this happens is the label for form's inputs.

Reply:
> An example where this happens is the label for form's inputs.

This is because label don't use `<Text />` component.